### PR TITLE
Updating About window and adding a new contributors category

### PR DESCRIPTION
**Pull request by TrechNex**
Still puzzling out how to add a new tab in About > Credits. 

I've left the version unchanged for now as I'm guessing that needs to be set in the build system, but we should preserve the `GIMP_VERSION` variable for doc purposes so we can tell users which version this is based on.

### CODE REVIEW

**Comment by TreacherousNexus** [comment]
The reason so many files are included was because I changed some translation files to say "Glimpse is brought to you by" instead of "GIMP is brought to you by"

**Comment by TrechNex** [comment]
There's probably enough here to squash and merge, but when I get time over the weekend I want to setup a new build environment in an actual Linux environment and use GTK Inspector tool to figure out where the different menu items are triggered from & built in the code.

https://wiki.gnome.org/action/show/Projects/GTK/Inspector?action=show&redirect=Projects%2FGTK%2B%2FInspector

File: authors.xml [hunk 1]
```diff
@@ -8,8 +8,12 @@
 
 <glimpse-authors xmlns="http://purl.org/dc/elements/1.1/">
 
-  <creator role="author" last-active="1.0">Bobby Moss</creator>
+  <creator role="author" last-active="1.0">Bobby Moss (TrechNex)</creator>
 
   <maintainer role="author" last-active="1.0">Clipsey</maintainer>
+  <maintainer role="author" last-active="1.0">Brainblasted</maintainer>
```
Comment (BrainBlasted) [inline authors.xml]: Can you use my name here? (Christopher Davis)

Comment (TrechNex) [reply authors.xml]: Now fixed! Sorry for the delay - wanted to rebuild and check everything looked okay from UI before marking as resolved.

**Pull request merged**